src/main/kotlin/com/autonomousapps/tasks/GenerateBuildHealthTask.kt

```diff
@@ -72,7 +75,8 @@ abstract class GenerateBuildHealthTask : DefaultTask() {
           val report = ProjectHealthConsoleReportBuilder(
             projectAdvice = projectAdvice,
             dslKind = dslKind.get(),
-            dependencyMap = dependencyMap.get().toLambda()
+            dependencyMap = dependencyMap.get().toLambda(),
+            useTypesafeProjectAccessors = useTypesafeProjectAccessors.get()
```

Comment:
nit: missing trailing comma (I can see it was already missing it, but let's improve it 😅 )